test/puppetlabs/trapperkeeper/services_test.clj

```diff
@@ -62,6 +65,55 @@
           app (bootstrap-services-with-empty-config [service1])]
       (is (not (nil? app))))))
 
+(deftest test-restart-file-correct-count
+  (testing "create a restart file and check that it correctly increments when a service starts"
+    (let [service1 (service Service1
```

Comment:
kitchensink has a temp-file function that may be helpful here. in the let block, you can assign the results of a `ks/temp-file` call to a `restart-file` var, and then use it below in your service configs. Then you don't need to worry about deleting it when the test is over.

That should also remove the need to require the fs library for your tests.